src/agent/main.c

```diff
@@ -15,20 +15,23 @@ const struct option options[] = { { ARG_HOST, required_argument, 0, ARG_HOST_SHO
                                   { ARG_ADDRESS, required_argument, 0, ARG_ADDRESS_SHORT },
                                   { ARG_NAME, required_argument, 0, ARG_NAME_SHORT },
                                   { ARG_CONFIG, required_argument, 0, ARG_CONFIG_SHORT },
+                                  { ARG_USER, no_argument, 0, ARG_USER_SHORT },
                                   { ARG_HELP, no_argument, 0, ARG_HELP_SHORT },
                                   { NULL, 0, 0, '\0' } };
 
-#define OPTIONS_STR \
-        ARG_PORT_SHORT_S ARG_HOST_SHORT_S ARG_ADDRESS_SHORT_S ARG_HELP_SHORT_S ARG_CONFIG_SHORT_S ARG_NAME_SHORT_S
+#define OPTIONS_STR                                                                               \
+        ARG_PORT_SHORT_S ARG_HOST_SHORT_S ARG_ADDRESS_SHORT_S ARG_HELP_SHORT_S ARG_CONFIG_SHORT_S \
+                        ARG_NAME_SHORT_S ARG_USER_SHORT_S
 
 static const char *opt_port = 0;
 static const char *opt_host = NULL;
 static const char *opt_address = NULL;
 static const char *opt_name = NULL;
 static const char *opt_config = NULL;
+static bool opt_user = false;
 
 static void usage(char *argv[]) {
-        printf("Usage: %s [-H host] [-p port] [-a address] [-c config] [-n name]\n", argv[0]);
+        printf("Usage: %s [-H host] [-p port] [-a address] [-c config] [-n name] [--user]\n", argv[0]);
```

Comment:
I think `doc/man/hirte-agent.1.md` needs to be updated as well and add the --user to the options.